plugin.video.ytchannels/plugin.video.ytchannels/resources/settings.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
Ups sorry, I missed this one. All the strings that are hardcoded in english need to be also localized